### ISSUE TITLE
Better error reporting/handling when importing items

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#198]
When adding items to an exhibit from SDR, if you include unpublished druids in the list of items the process will fail (even for the published druids in the list, I believe) without any indication of the failure provided to the user.

Ideally, the process would:

- Skip unpublished items but still add published items
- Show an error message so the exhibit curator knows how many (and ideally, which) druids were not added

At the least, showing the error message would be helpful in letting the curator know to double-check the list of druids.

**Comment by jcoyne** [comment]
How can I find an unpublished druid for testing purposes?

**Comment by ggeisler** [comment]
Argo? Otherwise maybe @caaster can help when she's back next week.

**Comment by caaster** [comment]
Here is one druid for the moment -- I am working to find a few more that are in different workflow statuses: druid:bp264pp3784 – registered status & will stay in this status until at least 29 July 2016

**Comment by caaster** [comment]
Here are two more druids. These are both in a test collection (I spoke to Lynn about using them -- all ok) -- so their workflow state should stay unchanged for the duration of our current work cycle.  druid:ym601hw1660 - in accessioning (described) status & druid:db473ds0563 - unknown status. NOTE: these are valid statuses that I run into a lot when trouble shooting in Argo.

**Comment by mjgiarlo** [comment]
@ggeisler @caaster We discussed this issue today and decided that design is needed when a *collection* that includes unpublished items is imported. We deprioritized this issue with the assumption that there wouldn't be cycles to get that done. I am splitting off a high-priority issue to handle only the case where a druid (or other string) that is unpublished is plugged into the form.

**Comment by ggeisler** [comment]
See the https://github.com/sul-dlss/exhibits/tree/sdr-items-status-mockup branch for a working design mockup. Discussed with @cbeer who will take over from here.

**Comment by ggeisler** [comment]
@caaster Nope. That link was to a code branch, not a static mockup. As mentioned in my comment, Chris agreed to take it from there. I can't speak to the status of it since then.
